packages/app/src/components/FileMatchIndicator.vue

```diff
@@ -3,9 +3,27 @@
     class="inline-flex items-center"
     data-cy="file-match-indicator"
   >
-    <span class="rounded font-medium bg-jade-100 h-24px text-center px-8px text-jade-600 truncate select-none">
+    <span
+      class="rounded font-medium  h-24px text-center px-8px  truncate select-none"
+      :class="color"
+    >
       <slot /></span>
   </div>
 </template>
+
 <script lang="ts" setup>
+import { computed } from 'vue'
+
+const props = withDefaults(defineProps<{
+  variant?: 'default' | 'empty'
+}>(), { variant: 'default' })
+
+const colorClasses = {
+  default: 'bg-jade-100 text-jade-600',
+  empty: 'bg-purple-100 text-purple-600',
```

Comment:
We should rename this prop, `empty` is confusing. What do you think about `info`?

Reply:
Sounds great! gonna rename it!